### DISCUSSION TITLE
Add Drops property

### DISCUSSION THREAD

**Discussion by ucosatti** [malzariey/filament-daterangepicker-filter#28] [Ideas]
It would be great to add the drops property, to set the picker should appears in relation of the HTML element.
Here is an example using a Modal form, and it can't reach its action buttons ( they out of screen).

<img width="1225" alt="image" src="https://github.com/malzariey/filament-daterangepicker-filter/assets/126919625/27eed288-d0f7-4fc5-a05b-c151d6dfcf1f">

**Comment by malzariey** [comment]
A PR is always welcomed.

**Comment by malzariey** [comment]
Available in version 2.5
